ch02_Context.md

```diff
@@ -132,7 +133,7 @@ kunnen gepubliceerd worden als BOMOS Aanvullende Module.
 
 ## Definities
 
-**Beheer en Ontwikkelen van standaarden (kortweg: beheer)**
+Beheer en Ontwikkelen van standaarden (kortweg: beheer)</dfn>
```

Comment:
Moet hier nog een `<dfn>` aan het begin of moet hij aan het eind weg worden gehaald?